### PR TITLE
Added coordinate transforms to FloatTransform2D.

### PR DESCRIPTION
**Pull request by davidpiuva**
Copied from FloatTransform3D with redundant depth scaling removed.

### CODE REVIEW

**Comment by davidpiuva** [comment]
@niklasborwell Peer review?

File: source/geometry/FloatTransform2D.ooc [hunk 1]
```diff
@@ -184,4 +185,20 @@ FloatTransform2D: cover {
 	createSkewingY: static func (angle: Float) -> This { This new(1.0f, angle sin(), 0.0f, 1.0f, 0.0f, 0.0f) }
 	createReflectionX: static func -> This { This new(-1.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0.0f) }
 	createReflectionY: static func -> This { This new(1.0f, 0.0f, 0.0f, -1.0f, 0.0f, 0.0f) }
+	referenceToNormalized: func ~float (imageSize: FloatVector2D) -> This {
+		toReference := This createScaling(imageSize x / 2.0f, imageSize y / 2.0f)
+		toNormalized := This createScaling(2.0f / imageSize x, 2.0f / imageSize y)
+		toNormalized * this * toReference
+	}
```
Comment (davidpiuva) [inline source/geometry/FloatTransform2D.ooc]: Modified from
```ooc
referenceToNormalized: func ~float (imageSize: FloatVector2D) -> This {
	toReference := This createScaling(imageSize x / 2.0f, imageSize y / 2.0f, 1.0f)
	toNormalized := This createScaling(2.0f / imageSize x, 2.0f / imageSize y, 1.0f)
	toNormalized * this * toReference
}
```

File: source/geometry/FloatTransform2D.ooc [hunk 2]
```diff
@@ -184,4 +185,20 @@ FloatTransform2D: cover {
...
+	referenceToNormalized: func ~int (imageSize: IntVector2D) -> This {
+		this referenceToNormalized(imageSize toFloatVector2D())
+	}
+	normalizedToReference: func ~float (imageSize: FloatVector2D) -> This {
+		toReference := This createScaling(imageSize x / 2.0f, imageSize y / 2.0f)
+		toNormalized := This createScaling(2.0f / imageSize x, 2.0f / imageSize y)
+		toReference * this * toNormalized
+	}
```
Comment (davidpiuva) [inline source/geometry/FloatTransform2D.ooc]: Modified from
```ooc
normalizedToReference: func ~float (imageSize: FloatVector2D) -> This {
	toReference := This createScaling(imageSize x / 2.0f, imageSize y / 2.0f, 1.0f)
	toNormalized := This createScaling(2.0f / imageSize x, 2.0f / imageSize y, 1.0f)
	toReference * this * toNormalized
}
```

**Comment by niklasborwell** [comment]
Sorry for the delay. Looks good.

**Pull request merged**